Microsoft.Toolkit.Uwp.UI.Controls/HeaderedContentControl/HeaderedContentControl.cs

```diff
@@ -119,7 +119,13 @@ private void SetHeaderVisibility()
         {
             if (GetTemplateChild(PartHeaderPresenter) is FrameworkElement headerPresenter)
             {
-                headerPresenter.Visibility = Header != null
+                var isEmptyString = false;
+                if (Header is string headerText)
+                {
+                    isEmptyString = string.IsNullOrEmpty(headerText);
```

Comment:
Let's just set the header visibility here and return, then we don't have to store a variable and do more logic.
```suggestion
                    headerPresenter.Visibility = string.IsNullOrEmpty(headerText) ? Visibility.Collapsed : Visibility.Visible;
                    return;
```

Reply:
Its ok? sooo coool